.github/actions/docker-compose-app.yml

```diff
@@ -3,13 +3,13 @@ version: "3.5"
 services:
   app:
     container_name: "app"
-    image: "ghcr.io/glpi-project/${PHP_IMAGE:-githubactions-php}"
+    image: "ghcr.io/glpi-project/${PHP_IMAGE:-githubactions-php-apache}"
```

Comment:
I recently (actually probably a month or two) started getting:
"ERROR: for app  manifest unknown"
when launching tests locally. It is still the case with the new app image. I believe it is caused by there not being any "latest" tag. I've had to manually change the file to specify one of the tags that exists.

Reply:
I put a default version to prevent this issue.